packages/lesswrong/components/linkPreview/HoverPreviewLink.tsx

```diff
@@ -119,6 +119,12 @@ const HoverPreviewLink = ({ href, contentSourceDescription, id, rel, noPrefetch,
           {children}
         </Components.ManifoldPreview>
       }
+
+      if (linkTargetAbsolute.host === "fatebook.io" || linkTargetAbsolute.host === "www.fatebook.io") {
```

Comment:
Does this also need to be added to `allowedIframeHostnames` in "lib/vulcan-lib/utils.ts" (a lot of the other embeds here are in that list, and this also seems to be using an iframe).

Reply:
ah yes, thanks

Reply:
actually, I'm not sure (because it was working before), but, I'd also kinda like to get Fatebook Embeds working.